### PR TITLE
completed final styling

### PR DESCRIPTION
**Pull request by shuhad786**
- Added styling to The UI.
- Added a touch-and-feel look to the UI.

### CODE REVIEW

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
### **Please consider the suggested changes below**

File: src/App.css [hunk 1]
```diff
@@ -1,87 +1,168 @@
...
+  font-weight: bold;
+  background-color: rgba(0, 191, 224, 0.67);
+  color: black;
+  padding: 0.3em;
+  box-shadow: 1px 2px 4px rgba(0, 191, 224, 0.67), 0 0 8px rgba(0, 191, 224, 0.67);
+}
+
+.displayNone {
+  display: none;
 }
```
Comment (CollinsTatang) [inline src/App.css]: Great job so far @shuhad786 

- [ ] Please your styling should be EXACTLY the same as the one provided on [Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
### **Take a Look at the required Design**
givenDesign
### **Take a Look at your Design**
yourOwnDesign

**Review by CollinsTatang — COMMENTED** [review]
Hi @shuhad786 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @shuhad786, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions
- [ ] It should have been stated by the previous reviewer that we have a newly updated `Readme` file. For new projects kindly use it as it is a general requirement
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**